server/channels/api4/plugin_test.go

```diff
@@ -2011,7 +2055,9 @@ func TestInstallMarketplacePluginPrepackagedDisabled(t *testing.T) {
 				var out []byte
 				out, err = json.Marshal(mPlugins)
 				require.NoError(t, err)
-				res.Write(out)
+				if _, err = res.Write(out); err != nil {
```

Comment:
![image](https://github.com/user-attachments/assets/31054536-c551-4628-a7ab-513289dab3f5)
Getting this

Reply:
Sorry, my comment was not precise enough. I meant removing the `if` clause entirely.

Instead of 

```golang
if _, err = res.Write(out); err != nil {
	require.NoError(t, err)
}
```

use
```golang
require.NoError(t, err)
```

Reply:
![image](https://github.com/user-attachments/assets/3211a4b0-55f1-4913-b1d1-d7f0d44fe319)
So, ultimately you're telling me to remove the checks? `cause `require.NoError(t, err)` is already above the `res.Write(xyz)` calls!

Reply:
The code snipped I previously shared was incomplete. Fixed that! I hope it's more clear now.

Reply:
I did the same changes a few hours ago, just waiting for your confirmation. Pushing it in a few mins!

Reply:
Fixed!